### PR TITLE
[bitnami/mongodb] Allow disabling the MongoDB Arbiter

### PR DESCRIPTION
**Pull request by stephan2012**
**Description of the change**

Allow disabling the MongoDB Arbiter

**Benefits**

Unconditionally deploying an Arbiter in a three Availability Zone situation creates a Primary-Secondary-Arbiter setup which usually is not desirable if the third location offers the possibility to run a full instance. See https://docs.mongodb.com/manual/reference/read-concern-majority/.

Also, the Chart deploys two StatefulSets, one for the MongoDB instance and one for the MongoDB Arbiter. When two MongoDB instances and one Arbiter are deployed, a Chart update causes the Pods for one instance and the arbiter to go down simultaneously rendering the MongoDB unavailable. This is probably not what you want when deploying a ReplicaSet.

**Possible drawbacks**

None. Change is backward compatible and Arbiter deployment needs to be disabled explicitly.

**Applicable issues**

**Additional information**

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [X] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/mongodb/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: mongodb
-version: 8.1.3
+version: 8.1.4
```
Comment (andresbono) [inline bitnami/mongodb/Chart.yaml:3]: Could you bump the minor one version? As this is a new feature
```suggestion
version: 8.2.0
```

**Comment by andresbono** [comment]
Thanks so much for your contribution @stephan2012! Could you please check my comments and let us know your thoughts?

File: bitnami/mongodb/Chart.yaml (hunk 1 above)
Comment (stephan2012) [reply bitnami/mongodb/Chart.yaml:3]: Sure.

**Comment by stephan2012** [comment]
> Can you also modify the description of the architecture in the README?

Done. Please review and feel free to adjust.

> Does it make sense for you to create a new helper? Let's say `mongodb.arbiter.enabled` which is defined like
> 
> ```
> {{- if and (eq .Values.architecture "replicaset") .Values.arbiter.enabled }}
>     {{- true -}}
> {{- else -}}
> {{- end -}}
> ```

Well, I do not find it _that_ benefical, but why not. Done.

**Comment by andresbono** [comment]
Thanks @stephan2012. We have made some minor changes. We are going to test it through our CI/CD and the PR will be merged.

**Comment by andresbono** [comment]
Thanks for your contribution, @stephan2012. The PR is now merged.

**Pull request merged**